### PR TITLE
tinygobb: update and make it work again

### PR DESCRIPTION
**Pull request by rminnich**
reduce the number of functions tinygobb updates
to just runone.

Add a placeholder cmds/exp/runone_tinygo.go,
so that gobusybox will not get confused.

### CODE REVIEW

**Comment by tlaurion** [comment]
Eli5: how to use this?

**Comment by rminnich** [comment]
I will add a readme.md

**Comment by tlaurion** [comment]
> I will add a readme.md

@rminnich gentle reminder. I would be curious to see reduction of size and properly test this

**Comment by tlaurion** [comment]
@rminnich just daw your talk https://osfc.io/2023/talks/tamago-and-tinygo-working-with-firmware-and-kernel-written-in-go/

Doesn't explain the usage, but explain the why's.
Good stuff!
